lib/src/feature/authentication/view/landing_screen.dart

```diff
@@ -61,9 +62,9 @@ class _LandingScreenState extends ConsumerState<LandingScreen> with SingleTicker
 
   @override
   Widget build(BuildContext context) {
-    final _controller = ref.read(AuthenticationController.controller);
+    final AuthenticationController _controller = ref.read(AuthenticationController.controller);
 
-    ref.listen(AuthenticationController.provider, (_, state) {
+    ref.listen(AuthenticationController.provider, (_, Object? state) {
```

Comment:
Try to be as specific as possible with your types, you should almost never use Object. In this case for example pretty sure this provides a BaseState so change the code to:

` ref.listen<BaseState>(AuthenticationController.provider, (_, BaseState? state) {`